collect_app/src/test/java/org/odk/collect/android/formlists/blankformlist/BlankFormListViewModelTest.kt

```diff
@@ -81,7 +80,7 @@ class BlankFormListViewModelTest {
     private val context = ApplicationProvider.getApplicationContext<Application>()
     private val syncRepository: SyncStatusAppState = mock()
     private val formsUpdater: FormsUpdater = mock()
-    private val scheduler = FakeScheduler()
+    private val scheduler = TestScheduler()
```

Comment:
`FakeScheduler` is great if you need control over background tasks (postpone running them to check something before and then after). In this test I don't need it so I think using a simpler solution (according to https://developer.android.com/kotlin/coroutines/test) is better here.
Another thing is that `FakeScheduler` would need to be improved for this tests because when `BlankFormListViewModel` is created two background tasks are started (see https://github.com/getodk/collect/pull/5077/files#diff-753aa7ab27aa86da15dd4c38ca810b17bc884db22cec003d19c2ad5451a5c523R84)
but `FakeScheduler` keeps only one and once another one is started the first one is removed:
https://github.com/getodk/collect/blob/master/testshared/src/main/java/org/odk/collect/testshared/FakeScheduler.kt#L17

any thoughts?

Reply:
Done.